src/context/events-context.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, createContext, useEffect } from "react";
+
+import {
+  createEventRequest,
+  listEventsRequest,
+} from "../respositories/events-repository";
+
+export const EventsContext = createContext({});
+
+export const EventsProvider = ({ children }) => {
+  const [createdEvent, setCreatedEvent] = useState({});
+  const [events, setEvents] = useState([]);
+
+  const createEvent = async (
+    inputTitle,
+    inputPrice,
+    inputDate,
+    inputDescription,
+    inputToken
+  ) => {
+    const fetchedEvent = await createEventRequest(
+      inputTitle,
+      inputPrice,
+      inputDate,
+      inputDescription,
+      inputToken
+    );
```

Comment:
5 params to a method is in the limit, I'd use a destructured object o something like that